src/controller/custom_controller.py

```
@@ -1,15 +1,33 @@
+import pdb
+
 import pyperclip
 from typing import Optional, Type
 from pydantic import BaseModel
 from browser_use.agent.views import ActionResult
 from browser_use.browser.context import BrowserContext
 from browser_use.controller.service import Controller, DoneAction
+from main_content_extractor import MainContentExtractor
```

Ruff (0.8.2)

`main_content_extractor.MainContentExtractor` imported but unused

Remove unused import: `main_content_extractor.MainContentExtractor`

(F401)